src/app/modules/main/chat_section/model.nim

```diff
@@ -23,7 +23,7 @@ type
 QtObject:
   type
     Model* = ref object of QAbstractListModel
-      items: seq[Item]
+      items*: seq[Item]
```

Comment:
@osmaczko exposing private list of items this way somehow breaks the rules we set for the new architecture. That may allow someone to unintentionally set or update the list (`items`) without emitting appropriate signal and bring some inconsistencies here. Also exposing private members is hard to follow/maintain in a big project. So please revert this.

Reply:
https://github.com/status-im/status-desktop/pull/4727#discussion_r801024464